salt_stack/salt/build-lambda/init.sls

```diff
@@ -0,0 +1,71 @@
+# Setup the lambda development/build server.
```

Comment:
This looks like it was based on an old Salt formula? I updated `salt_stack/salt/lambda-dev/init.sls` for Python 3.6.